user-profiles.md

```diff
@@ -0,0 +1,36 @@
+# User Profiles
+
+The purpose of this document is to aid in the development of Helm features by evaluating Helm features against those who will be using them.
+
+When considering requirements and implementation solutions it's useful look at who uses Helm to have a better understanding of how something should work. The Helm user profiles describe the different types of users and contributors to Helm along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
+
+## Preface
+
+* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories.
+* Kubernetes Cluster Operators are out of scope for this document. A cluster operator is one who manages the operation of a Kubernetes cluster that applications can run in.
+
+## Profiles
+
+Profiles describe a type of role a user may perform. A real person may perform more than one role and have more than one profile apply to them. How this mapping works between profiles and real people can vary between companies and other organizations. To handle this variation we focus on the user profiles rather than how they may map to people in these different organizations.
+
+### 1. Application Operator
+
+Application operators take an application and operate it within a Kubernetes cluster. For example, the operation of WordPress and MySQL. This is not to be confused with the role of a Kubernetes cluster operator (see the preface above for more detail).
```

Comment:
I noticed above that we capitalize "Kubernetes Cluster Operator". Might be good to make it all consistent and either "Capitalize The Whole Title" or just "Kubernetes". :)

Same thing with "Application Operator" and other titles used in this doc, e.g. "application developer" on line 26